validator.go

```diff
@@ -285,17 +285,15 @@ func (v *validator) verifySubject(claims Claims, cmp string, required bool) erro
 func errorIfFalse(value bool, err error) error {
 	if value {
 		return nil
-	} else {
-		return err
 	}
+	return err
```

Comment:
The rest of the changes seem fine to me. I kind of like the switch better than the old if. Thanks for that, but in the `errorIfFalse` and `errorIfRequired` functions I would prefer to keep the explicit else branch for better readability.

Reply:
reverted